netlify.toml

```diff
@@ -3,10 +3,6 @@ base = "docs/"
 command = "yarn run build"
 publish = "build/"
 
-# Adding redirects - to be deleted when landing page is ready
-[[redirects]]
-from = "/"
```

Comment:
Personally, I think it's in a nice shape to start using. It has all our new brand elements and messaging ideas, which we don't have anywhere else right now. It's also a much better starting point for new users because it gives a good overview of Nebari (compared to the current docs homepage where the redirect goes). 

We can keep making updates to it. :)

Reply:
Agree! While not perfect this is way better than anything we've had in the past.

AFAIK  the remaining changes can be added iteratively even if the landing page goes live, namely:

-  Updating the footer
- adding background images 
- deal with font size and the such in a more consistent manner 

Am I missing anything else? 

WDYT @gabalafou?

Reply:
Yeah I think that could be okay. There a little funkiness in the responsiveness. For example, below the default Docusaurus breakpoint the multi-column grid of logos under "Integrations" collapses to two columns. But they're not centered. And there are probably a few other blemishes like that.